src/utils/__tests__/datetime-test.js

```diff
@@ -160,4 +161,30 @@ describe('utils/datetime', () => {
             expect(result.toISOString()).toBe('2018-06-13T00:00:00.000Z');
         });
     });
+
+    describe('convertISOStringtoRFC3339String()', () => {
+        test.each([
+            ['2018-06-13T00:00:00.000Z', '2018-06-13T00:00:00.000Z'],
+            ['2018-06-13T00:00:00.000-05', '2018-06-13T00:00:00.000-05:00'],
+            ['2018-06-13T00:00:00.000-0500', '2018-06-13T00:00:00.000-05:00'],
+            ['2018-06-13T00:00:00.000-05:00', '2018-06-13T00:00:00.000-05:00'],
+        ])('should convert %s to %s correctly', (from, to) => {
+            const input = convertISOStringtoRFC3339String(from);
+
+            expect(input).toEqual(to);
+        });
+
+        test.each([
+            ['bad date'],
+            [''],
+            ['2018-01-01'],
+            ['2019-01-02T00:00:00'],
+            ['2019-01-03T00:00:00.000'],
+            ['2019-01-04T00:00:00.000z'],
+        ])('should throw on bad date format: %s', () => {
```

Comment:
It doesn't look like the parameterized value is used in the test.

Reply:
good catch!